lib/item.rb

```diff
@@ -0,0 +1,3 @@
+class Item
+  attr_accessor :id, :name, :unit_price, :qty
+end
```

Comment:
If a class is just a collection of values, and it doesn't have behaviour, it's normally better to use a Hash.